packages/lit-html/src/lit-html.ts

```diff
@@ -523,18 +536,22 @@ const getTemplateHtml = (
           (attrNameEndIndex === -2 ? (attrNames.push(undefined), i) : end);
   }
 
+  let htmlResult: string|TrustedHTML =
+      html + (strings[l] || '<?>') + (type === SVG_RESULT ? '</svg>' : '');
+  if (policy) {
+    htmlResult = policy.createHTML(htmlResult);
+  }
+
   // Returned as an array for terseness
   return [
-    // We don't technically need to close the SVG tag since the parser will
-    // handle it for us, but the SSR parser doesn't like that.
-    // Note that the html must end with a node after the final expression to
-    // ensure the last ChildPart has an end node, hence adding a comment if the
-    // last string was empty.
-    html + (strings[l] || '<?>') + (type === SVG_RESULT ? '</svg>' : ''),
+    htmlResult as TrustedHTML,
     attrNames,
   ];
 };
 
+
+const EMPTY_TEXT = trustedTypes?.emptyScript as unknown as '' ?? '';
```

Comment:
I would go ahead and use `||` here, since this is only run once on startup. Optimize for bytes.

Reply:
Switched to a ternary